gramex/handlers/mlhandler.py

```diff
@@ -198,7 +198,7 @@ def _predict(self, data=None, score_col=''):
 
     def _check_model_path(self):
         try:
-            self.model = self.store.load_model()
+            self.model = get_model(self.store.model_path, {})
```

Comment:
This is a problem: `get_model` can only raise `ImportError`s, not  `FileNotFoundError`s, making the following two lines unreachable. Breaks three tests.

Reply:
This function should not exist, move this logic to the `ModelStore`.